ride_share.rb

```diff
@@ -0,0 +1,185 @@
+DRIVERS_INFO = [
+  {
```

Comment:
Consider how making your top level data structure a hash with the `driver_id`s as keys could simplify your code just a bit.